sal_interfaces/MTAOS/MTAOS_Commands.xml

```diff
@@ -3,244 +3,159 @@
+    <Description>Run OFC on the latest wavefront errors data. Before running this command, you must have ran runWEP at least once. This command will run ofc to compute correction but won't apply them. Use issueCorrection to apply the corrections. This allow users to evaluate whether the corrections are sensible before applying them.</Description>
     <item>
       <EFDB_Name>userGain</EFDB_Name>
       <Description>Gain value between 0 and 1. Set to -1 to ignore user gain.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-  </SALCommand>
-  <SALCommand>
-    <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_command_processCmosWavefrontError</EFDB_Topic>
-    <Description>Process the wavefront data collection by CMOS camera.</Description>
-    <item>
-      <EFDB_Name>intraImgPath</EFDB_Name>
-      <Description>Absolute path to intra-focal wavefront image FITS file.</Description>
-      <IDL_Type>string</IDL_Type>
-      <IDL_Size>1</IDL_Size>
+      <IDL_Type>float</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>extraImgPath</EFDB_Name>
-      <Description>Absolute path to extra-focal wavefront image FITS file.</Description>
+      <EFDB_Name>config</EFDB_Name>
+      <Description>Yaml configuration string with additional parameters to be passed to OFC. If options are invalid command will be rejected. If not given, use the standard configuration.</Description>
       <IDL_Type>string</IDL_Type>
-      <IDL_Size>1</IDL_Size>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>MTAOS</Subsystem>
+    <EFDB_Topic>MTAOS_command_addAberration</EFDB_Topic>
+    <Description>Utility command to add aberration to the system based on user provided wavefront errors. The command assume uniform aberration on all sensors.</Description>
     <item>
-      <EFDB_Name>sglBrightStarPixelX</EFDB_Name>
-      <Description>Single bright star pixel x position. Put -99999 for the empty star.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>unitless</Units>
-      <Count>10</Count>
-    </item>
-    <item>
-      <EFDB_Name>sglBrightStarPixelY</EFDB_Name>
-      <Description>Single bright star pixel y position. Put -99999 for the empty star.</Description>
+      <EFDB_Name>wf</EFDB_Name>
+      <Description>Array with z4-z22 terms of wavefront errors. Indexes are considered to be ANSI (https://en.wikipedia.org/wiki/Zernike_polynomials#OSA/ANSI_standard_indices).</Description>
       <IDL_Type>double</IDL_Type>
-      <Units>unitless</Units>
-      <Count>10</Count>
+      <Units>micron</Units>
+      <Count>19</Count>
     </item>
     <item>
-      <EFDB_Name>filter</EFDB_Name>
-      <Description>Active filter type. Use 1-6 for u, g, r, i, z, and y. Use 7 for reference.</Description>
-      <IDL_Type>int</IDL_Type>
+      <EFDB_Name>userGain</EFDB_Name>
```

Comment:
What should be the order of this command and `MTAOS_command_runOFC()`? Are they independent? If not, what will happen if the users put two different `userGain` values?

Reply:
I think this made it here by (cut and paste) mistake... I'll remove it.

Reply:
removed